app/lib/theme/subthemes/icon_button_theme.dart

```diff
@@ -7,31 +7,29 @@ class FlexIconButtonTheme {
 
   static IconButtonThemeData lightIconButtonTheme = IconButtonThemeData(
     style: IconButton.styleFrom(
-      elevation: 1,
-      shadowColor: FlexAppColorScheme.lightScheme.primary,
-      foregroundColor: FlexAppColorScheme.lightScheme.primary,
+      elevation: 0,
       backgroundColor: FlexAppColorScheme.lightScheme.onPrimary,
+      foregroundColor: FlexAppColorScheme.lightScheme.primary,
       disabledForegroundColor: FlexAppColorScheme.lightScheme.disabled,
-      disabledBackgroundColor: FlexAppColorScheme.lightScheme.background,
-      padding: const EdgeInsets.all(FlexSizes.xs),
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(FlexSizes.borderRadiusSm),
+      disabledBackgroundColor: FlexAppColorScheme.lightScheme.onBackground,
+      padding: const EdgeInsets.all(FlexSizes.sm),
+      shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.zero,
```

Comment:
Think this will ever be round again? Seems silly to have roundedRectagle with 0 border radius..

Reply:
Good catch!